esp/src/eclwatch/WUQueryWidget.js

```diff
@@ -456,6 +457,13 @@ define([
             this.workunitsGrid.onSelectionChanged(function (event) {
                 context.refreshActionState();
             });
+
+            aspect.after(this.workunitsGrid, 'gotoPage', function (promise, args) {
```

Comment:
Is this correct?  According to the docs (https://dojotoolkit.org/reference-guide/1.10/dojo/aspect.html) it should be:
```
aspect.after(this.workunitsGrid, 'gotoPage', function(deferred){
  return deferred.then(function(args){
    args[0] > 1 ? context._idleWatcher.stop() : context._idleWatcher.start();
  });
});
```